public/main.css

```diff
@@ -0,0 +1,41 @@
+/* line 2, ../src/stylesheets/globals/_reset.scss */
```

Comment:
Also @erindepew -- now that I kind of understand what Sass is, what is the reason of having main.css (and the other rendered/compiled css files) checked into git?  Shouldn't we just be checking in the sass source files and having part of the standing up being a sass compile of scss --> css?

Or am I mis-understanding the relationship between scss and css files?